docs/Behaviour - Server Side.md

```diff
@@ -17,26 +17,26 @@ Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the
 ## State of Sender
 
 A Sender managed with IS-11 has the following states:
-- `Unconstrained` when Active Constraints of this Sender is an empty `constraint_sets` array.
-- `Constrained` when this Sender, the Flow and the `/transportfile` (if used) associated with this Sender and the Source associated with this Flow satisfy the Active Constraints.
-- `Active Constraints Violation` when this Sender, the Flow or the `/transportfile` (if used) associated with this Sender or the Source associated with this Flow does not satisfy the Active Constraints. When an inactive Sender is in this state, it MUST NOT allow [IS-05][IS-05] activations.
-- `No Signal` when there is no signal from Inputs associated with this Sender.
-- `Awaiting Signal` when Active Constraints was just PUT and the signal from Inputs associated with this Sender is not stable yet. This is a transitional state until one of the previous ones can be established.
+- `unconstrained` when Active Constraints of this Sender is an empty `constraint_sets` array.
+- `constrained` when this Sender, the Flow and the `/transportfile` (if used) associated with this Sender and the Source associated with this Flow satisfy the Active Constraints.
+- `active_constraints_violation` when this Sender, the Flow or the `/transportfile` (if used) associated with this Sender or the Source associated with this Flow does not satisfy the Active Constraints. When an inactive Sender is in this state, it MUST NOT allow [IS-05][IS-05] activations.
+- `no_signal` when there is no signal from Inputs associated with this Sender.
+- `awaiting_signal` when Active Constraints was just PUT and the signal from Inputs associated with this Sender is not stable yet. This is a transitional state until one of the previous ones can be established.
 
 ## State of Receiver
 
 A Receiver managed with IS-11 has the following states:
-- `No Transport File` when there's no active `transport_file`.
-- `OK` when this Receiver's active `transport_file` does not violate Receiver Capabilities.
-- `Receiver Capabilities Violation` when this Receiver's active `transport_file` violates Receiver Capabilities.
+- `no_transport_file` when there's no active `transport_file`.
+- `ok` when this Receiver's active `transport_file` does not violate Receiver Capabilities.
+- `receiver_capabilities_violation` when this Receiver's active `transport_file` violates Receiver Capabilities.
```

Comment:
```suggestion
- `unknown` when this Receiver is inactive or it's active and it's impossible to say whether the Flow is compliant with this Receiver (e.g. it's activated without a `transport_file`).
- `compliant_flow` when this Receiver is active and the Flow is compliant with this Receiver (e.g. there's an active `transport_file` and it does not violate Receiver Capabilities).
- `non_compliant_flow` when this Receiver is active and the Flow is non compliant with this Receiver (e.g. there's an active `transport_file` which violates Receiver Capabilities).
```

Reply:
I would add "Capabilities" to all 3 cases "compliant with this Receiver Capabilities"

Reply:
```suggestion
- `unknown` when this Receiver is inactive or it is active and it is impossible to say whether the Flow is compliant with Receiver Capabilities of this Receiver (e.g. it is activated without a `transport_file`).
- `compliant_flow` when this Receiver is active and the Flow is compliant with Receiver Capabilities of this Receiver (e.g. there is an active `transport_file` and it does not violate the Receiver Capabilities).
- `non_compliant_flow` when this Receiver is active and the Flow is non compliant with Receiver Capabilities of this Receiver (e.g. there is an active `transport_file` which violates the Receiver Capabilities).
```

Reply:
There is a pending issue about the name "flow" versus "stream" ... If we go with "stream" we'll have to do a global replacement of "flow" for "stream" ...